src/main/java/it/gov/pagopa/pu/debtpositions/repository/InstallmentNoPIIRepository.java

```diff
@@ -28,6 +31,15 @@ public interface InstallmentNoPIIRepository extends JpaRepository<InstallmentNoP
   @Query("UPDATE InstallmentNoPII i SET i.status = :status WHERE i.installmentId = :installmentId")
   void updateStatus(@Param("installmentId") Long installmentId, @Param("status") InstallmentStatus status);
 
+
+  @RestResource(exported = false)
+  @Transactional
+  @Modifying
+  @Query("UPDATE InstallmentNoPII i " +
+    "SET i.status = :status, i.syncStatus = i.syncStatus " +
+    "WHERE i.installmentId = :installmentId")
+  void updateStatusAndSyncStatus(@Param("installmentId") Long installmentId, @Param("status") InstallmentStatus status, @Param("syncStatus") InstallmentSyncStatus syncStatus);
```

Comment:
just call update status, sync status is a side effect of this operation
```suggestion
  void updateStatus(@Param("installmentId") Long installmentId, @Param("status") InstallmentStatus status, @Param("syncStatus") InstallmentSyncStatus syncStatus);
```

Reply:
ps: I see it already exists a `updateStatus`, but having a method to update the status without considering syncStatus is dangerous, could you verify if we could change that method and adequate the code that is actually invoking it?